linters/bandit.js

```diff
@@ -0,0 +1,68 @@
+// Copyright 2018 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+const cache = require('../cache')
+
+const report = require('../bandit/bandit_report')
+
+module.exports = class Bandit {
+  get name () {
+    return 'bandit'
+  }
+
+  /**
+   * The path to the generated report file
+   * relative to the working directory
+   */
+  get reportPath () {
+    return '../bandit.json'
```

Comment:
Looking into it

Reply:
Well I don't see change here.
Do you have other idea in mind or?

Reply:
I don't think the linter specific code should concern itself with the save location of the report path, this should entirely be controlled by the calling code (the app batch runner or a unit test file). I might actually drop the '../' prefix and move that elsewhere.
The only current use is factoring out the report path between the args of the spawned linter and the callback that reads it back.

Reply:
I agree that the caller should specify where to save the report.
Maybe you can do something like
``get reportPath (workingDirectory) { return path.join(workingDirectory, '../bandit.json') }``

Reply:
I'll see what I can do (it's tricky because getters cannot have parameters: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get)

Reply:
I am sorry. You really don't need to give workingDirectory to this get function.
Maybe you should rename the function and use it like this:
``get reportFileName () { return 'bandit.json') }``

Reply:
Done